run_unittests.py

```diff
@@ -3312,33 +3313,45 @@ def test_apple_bitcode_modules(self):
         self.build()
         self.run_tests()
 
-    def test_uninstalled_usage_external_library(self):
+    def test_usage_external_library(self):
         '''
         Test that uninstalled usage of an external library (from the system or
-        PkgConfigDependency) works. On Linux/BSD/macOS it tests if RPATHs are
-        set correctly.
-
-        TODO: On Windows, this can test whether PATH is set properly
+        PkgConfigDependency) works. On macOS, this workflow works out of the
+        box. On Linux, BSDs, Windows, etc, you need to set extra arguments such
```

Comment:
That this does not work on linux and BSDs sounds more like a bug in meson that needs to be fixed?
So i think that comment is misleading.

Reply:
No, it's not a bug. If you do not install your libraries into the runtime linker paths (ld.so.conf), you need to set the `install_rpath:` or your libraries won't be found.

Reply:
It says """Test that uninstalled usage of an external library [...] works""". So maybe that's not what it seems to say? Not sure.